### PR TITLE
Gallery Update: Split ML/CV into two sections, added Regression

### PR DESCRIPTION
**Pull request by xhlulu**
Two sections because ML section was getting crowded

### CODE REVIEW

**Comment by xhlulu** [comment]
Thanks! @shammamah Here's the deployed app: https://dash-docs-pr-189.herokuapp.com/

File: tutorial/gallery.py [hunk 1]
```diff
@@ -383,8 +385,21 @@ def AppSection(app_name,
...
-            ''',
-            width=12
+            '''
+        ),
+
+        AppSection(
+            app_name='Explore Linear Models',
+            app_link='https://dash-gallery.plotly.host/dash-regression',
+            code_link='https://github.com/plotly/dash-regression',
+            img_src='https://raw.githubusercontent.com/plotly/dash-docs/master/images/dash-regression.gif',
```
Comment (shammamah) [inline tutorial/gallery.py]: This image link is giving me a 404, and it's not showing up on the page - other than that, it looks good :)

**Comment by Marc-Andre-Rivet** [comment]
@xhlulu @shammamah Is this still active?

**Comment by xhlulu** [comment]
I think we can close this one for now.

**Pull request closed without merging**